### DISCUSSION TITLE
Anyone have experience migrating from hMailServer?

### DISCUSSION THREAD

**Discussion by wertercatt** [docker-mailserver/docker-mailserver#2437] [Q&A]
I was previously hosting my personal mail server on hMailServer, a Windows program. I've been able to successfully set my accounts up on docker-mailserver, but I'm having a bit of a critical issue. Namely, I'm unable to successfully migrate the stored emails from hMailServer. Using their backup tools, I was able to get a .7z containing the messages, but they're stored in a directory layout that won't just 'drop in' to the directory layout docker-mailserver uses. You can see the layout of the backup here: https://gist.github.com/wertercatt/18dba03be4adbd708ff5f5f68fab7b6f

**Comment by casperklein** [comment]
I am not sure, if this kind of migration is easily possible. If you use IMAP, you could try [this](https://github.com/imapsync/imapsync) tool.
